common/test/acceptance/pages/lms/teams.py

```diff
@@ -19,3 +28,27 @@ def is_browser_on_page(self):
     def get_body_text(self):
         """ Returns the current dummy text. This will be changed once there is more content on the page. """
         return self.q(css='.page-content-main').text[0]
+
+    def browse_topics(self):
+        """ View the Browse tab of the Teams page. """
+        self.q(css=BROWSE_BUTTON_CSS).click()
+
+    def get_topic_cards(self):
+        """Return a list of the topic cards present on the page."""
+        return self.q(css=TOPIC_CARD_CSS).results
+
+    def get_pagination_text(self):
+        """Return the text showing which topics the user is currently viewing."""
+        return self.q(css=PAGINATION_TEXT_CSS).text[0]
+
+    def get_current_page_text(self):
+        """Return the text showing the current page."""
+        return self.q(css=CURRENT_PAGE_TEXT_CSS).text[0]
+
+    def go_to_topics_list_page(self, page_number):
```

Comment:
I think that these methods should belong to a topic-specific `bok_choy` `PageObject`. That way we can have separate `PageObjects` for the "My Teams" and "Browse" subtabs. In that case, this method could be unambiguously named `go_to_page`.